src/BookNavigator/BookNavigator.js

```diff
@@ -308,12 +308,13 @@ export class BookNavigator extends LitElement {
   manageFullScreenBehavior(event) {
     this.emitFullScreenState(event);
 
-    const { isFullscreenActive } = this.bookreader;
+    const { isFullscreenActive, fullscreenScrollPos = {} } = this.bookreader;
+    const { scrollX = 0, scrollY = 0 } = fullscreenScrollPos;
 
     if (!isFullscreenActive) {
       this.fullscreenMgr.teardown();
     } else {
-      this.fullscreenMgr.setup();
+      this.fullscreenMgr.setup(scrollX, scrollY);
```

Comment:
We can get `$('.BookReader').offset()` (Or however we read the root BookNavigator element) to give us these values; that way we don't have to keep track of them in `fullscreenScrollPos` at all!

Reply:
I have an interesting dilemma here, when the `fullscreenActive` class gets appended, bc of position: fixed, the bookreader blinks out of view until the height is dynamically set.  this is causing side effects of scroll positions & offsets to effectively return `0` - so as a reaction, I passed it through an instance param (bc passing it through the custom event broke all the things 😞 )
I will retry with your suggestion 🤞

Reply:
really good catch. i've updated to reflect our discussion: on fullscreen start, FS Manager will set class.